### PR TITLE
Adding Askubuntu and playstore

### PR DESCRIPTION
**Pull request by viciousButterfly**
I have added askubuntu and Google playstore search engines.

### CODE REVIEW

File: src/searchor/__init__.py [hunk 1]
```diff
@@ -58,6 +59,7 @@ class Engine(Enum):
     NPM = "https://www.npmjs.com/search?q={query}"
     OpenLibrary = "https://openlibrary.org/search?q={query}"
     OperaAddons = "https://addons.opera.com/en/search/?query={query}"
+    Playstore = "https://play.google.com/store/search?q={query}"
```
Comment (ArjunSharda) [inline src/searchor/__init__.py:62]:
```suggestion
    GooglePlayStore = "https://play.google.com/store/search?q={query}"
```

File: src/searchor/__init__.py [hunk 2]
```diff
@@ -9,6 +9,7 @@ class Engine(Enum):
     AlternativeTo = "https://alternativeto.net/browse/search/?q={query}"
     Apple = "https://www.apple.com/search/{query}"
     Ask = "https://www.ask.com/web?q={query}"
+    Askubuntu = "https://askubuntu.com/search?q={query}"
```
Comment (ArjunSharda) [inline src/searchor/__init__.py:12]:
```suggestion
    AskUbuntu = "https://askubuntu.com/search?q={query}"
```

**Review by ArjunSharda — CHANGES_REQUESTED** [review]
Looks good to me, but can you please push the code to change the name to what i've suggested? Searchor's codebase follows the UpperCamelCase convention.

File: src/searchor/__init__.py (hunk 2 above)
Comment (viciousButterfly) [reply src/searchor/__init__.py:12]: Yes I'll

**Comment by viciousButterfly** [comment]
Added AskUbuntu and GooglePlayStore with the mentioned changes in the naming convention

File: src/searchor/__init__.py [hunk 3]
```diff
@@ -58,7 +60,7 @@ class Engine(Enum):
     NPM = "https://www.npmjs.com/search?q={query}"
     OpenLibrary = "https://openlibrary.org/search?q={query}"
     OperaAddons = "https://addons.opera.com/en/search/?query={query}"
-    PyPi = "https://pypi.org/search/?q={query}"
+    PyP = "https://pypi.org/search/?q={query}"
```
Comment (ArjunSharda) [inline src/searchor/__init__.py:63]:
```suggestion
    PyPi = "https://pypi.org/search/?q={query}"

**Review by ArjunSharda — APPROVED** [review]
LGTM! Will merge in the next latest update.

**Review by itskegnh — APPROVED** [review]

**Pull request merged**